node/src/main/scala/coop/rchain/node/api/package.scala

```diff
@@ -43,6 +43,7 @@ package object api {
             .bindService(deployGrpcService, grpcExecutor)
         )
         .addService(ProtoReflectionService.newInstance())
+        .compressorRegistry(null)
```

Comment:
We have compression defined here for the protocol transport. I'm not sure what this is doing?
https://github.com/rchain/rchain/blob/13b2c6e0ae90728375df44564251091c00b224eb/comm/src/main/scala/coop/rchain/comm/transport/Chunker.scala#L17-L25

Reply:
I think that's only for the casperMessage.
The deployService doesn't support that now.
They are two different services.

Reply:
And not for all CasperMessage but only for streamed messages blocks.

Reply:
I meant this just as an example. Can we enable this on deploy service or `compressorRegistry(null)` is doing this?

Reply:
`compressorRegistry(null)` should do the trick.